packages/mobile/src/screens/profile-screen/ProfilePicture.tsx

```diff
@@ -13,5 +17,17 @@ export const ProfilePicture = (props: Partial<ProfilePictureProps>) => {
     'updatedProfilePicture'
   ])
 
-  return <ProfilePictureBase profile={profile} {...props} />
+  return (
+    <Box
+      style={css({
+        position: 'absolute',
+        top: spacing.unit13,
+        left: spacing.unit3,
+        zIndex: 100
```

Comment:
do we need to register this in the `zIndex.ts` file?

Reply:
good looks, i moved it from a different file, but will use zIndex file as a refactor :)

Reply:
Yeah I never know how strict to be with zIndex, but given that we have it, feels like we should use it. Worst is to have an unreliable reference. However, I think there are spots where the usage is not globally relevant, so it's tough